internal/storage/db/internal/model.go

```diff
@@ -96,3 +97,16 @@ func (gpw *GeneratedPolicyWrapper) Scan(src interface{}) error {
 
 	return nil
 }
+
+type PolicyRevision struct {
+	RevisionID  int64 `db:"revision_id"`
+	ID          namer.ModuleID
+	Kind        string
+	Name        string
+	Version     string
+	Description string
+	Action      string
+	Disabled    bool
+	Definition  PolicyDefWrapper
+	Timestamp   time.Time `db:"update_timestamp"`
+}
```

Comment:
Isn't this dead code?

Reply:
I think it isn't needed for now. But it is somehow going to be used when reading the policy revision. Anyway, I can remove it if you want.